### ISSUE TITLE
Infrastructure - projected monthly log size is 10% over papertrail account limit

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#3524]
Last month we breached 1Gig total logs and papertrail stopped collecting log information. This is a problem as we need to go to each box to investigate log messages, that is pretty time consuming. 

We are on track so far to hit 1.1 to 1.2 gigs at the end of this month. For this ticket we need to either increase our monthly costs to increase our logging limit, or we need to find a way to log 20% less.

**Comment by prastle** [comment]
@DanVanAtta  I responded in the server thread with some more info.  We can up the payments for logging but I suspect if we fix the missing bots etc we will be fine.  Np bumping for a month or two tho.

**Comment by prastle** [comment]
@DanVanAtta  Well I will admit I had some password attempt failures on 600.  Since strangely It accepts my ssh but not admin.  The rest do.  Either way if you have time today to examine the bots and infra for jp and 600 that would be great.

**Comment by RoiEXLab** [comment]
@DanVanAtta I'm pretty sure most of the logs are being created by the update scripts that run every 5 minutes.
So making those less verbose should make a huge difference

**Comment by DanVanAtta** [comment]
We can also set them to run every 10 minutes as well. Reducing any verbosity IMO is a good step.

**Comment by DanVanAtta** [comment]
Several things were fixed here:
- fixed server in error. Not reporting stack trace every 5 helps : )
- we removed verbosity and some debug output.

Paper trail is reported a projected total of 888MB. Our target is to stay under 1GB to avoid a large price jump and stay in the lowest cost price bracket - so cheers that we are there!

Let's re-open or create a new ticket if we start to exceed the 1GB estimate.
